README.md

```diff
@@ -357,19 +357,28 @@ When triggered, Travis will build the project, then run the test scripts, and ou
 
 Steps in the automation can be set in `.travis.yml`, located in the root directory.
 
-### Releasing
+### Branching and Releasing
 
-To perform a release of any changed packages, run
+We are adopting a ["oneflow" branching strategy](https://www.endoflineblog.com/oneflow-a-git-branching-model-and-workflow) where:
 
-```
-lerna publish
-```
-
-More information about this command can be found in the README for [lerna publish](https://github.com/lerna/lerna/tree/master/commands/publish)
+- `develop` is an eternal branch - bleeding edge, reviewed but not necessarily released code
+- `master` is another eternal _pointer_ branch to the latest released production tag. we advance this branch after a successful release with `git checkout master && git merge --ff-only @ibm-watson/discovery-react-components@x.x.x` where `x.x.x` is the latest released production tag. example steps:
+  - `git checkout master`
+  - `git merge --ff-only @ibm-watson/discovery-react-components@1.0.1`
+- `release/x.x.x` is a temporary branch created for beginning a production release. this contains all the features needed for the release and will only receive bugfixes. Once the release is complete, this branch is tagged and merged back into `develop`. example steps:
+  - `git checkout release/2.3.0`
+  - if we want to publish a release candidate (not final build):
```

Comment:
unsure which situations we would want to publish a release candidate vs just doing testing on our develop branch?

Reply:
in case its not the final final release (i.e. we've cut a release branch and we haven't determined if it fixed the issue yet, we can cut an `rc` to pull it into tooling first and if all checks out, just run the same command again without the `rc` tag and it should do an official release)

Reply:
really sweet ok 👍